docs/arcaflow/getting-started.md

```diff
@@ -1,101 +1,97 @@
 
-## Step 2: Get the engine
+!!! note
+    The default container platform for the Arcaflow engine is Podman. To use another platform, a [configuration file](/arcaflow/running/setup/#configuration) is required.
 
-Head on over to the [GitHub releases page](https://github.com/arcalot/arcaflow-engine/releases) and download the latest release.
+A repository of [example workflows](https://github.com/arcalot/arcaflow-workflows) is available for reference and practice. Let's try running the [basic example](https://github.com/arcalot/arcaflow-workflows/basic-examples/basic).
 
-## Step 3: Create your first plugin
+First we will clone the example workflows repository and set our working directory:
 
-Let's create a simple hello-world plugin in Python. We'll publish the code here, you can find the details in the [Python plugin guide](plugins/python/first.md).
-
-```python title="plugin.py"
-#!/usr/local/bin/python3
-import dataclasses
-import sys
-from arcaflow_plugin_sdk import plugin
-
-
-@dataclasses.dataclass
-class InputParams:
-    name: str
-    
-    
-@dataclasses.dataclass
-class SuccessOutput:
-    message: str
+```bash
+git clone https://github.com/arcalot/arcaflow-workflows.git
+export WFDIR=arcaflow-workflows/basic-examples/basic/
+```
 
+Then we will run the workflow, setting the working directory as the context, and defining the workflow, configuration, and input files to use:
 
-@plugin.step(
-    id="hello-world",
-    name="Hello world!",
-    description="Says hello :)",
-    outputs={"success": SuccessOutput},
-)
-def hello_world(params: InputParams):
-    return "success", SuccessOutput(f"Hello, {params.name}")
+```bash
+arcaflow --context $WFDIR --workflow workflow.yaml --config config.yaml --input input.yaml
+```
 
+Arcaflow will display logs, depending upon the configured verbosity, and then will return the machine-readable output of the workflow in YAML format:
 
-if __name__ == "__main__":
-    sys.exit(
-        plugin.run(
-            plugin.build_schema(
-                hello_world,
-            )
-        )
-    )
+```bash
+output_data:
+    example:
+        message: Hello, Arcalot!
+output_id: success
 ```
 
-!!! tip
-    Further reading: [Creating your first Python plugin](plugins/python/first.md)
-
-## Step 4: Build the plugin
+[Learn more about running workflows &raquo;](/arcaflow/running/){ .md-button }
 
-Next, let's create a `Dockerfile` and build a container image:
+## Writing Workflows
+As a workflow author, you determine the steps of the workflow, how data will pass between the steps, what input is required from the workflow user, and what output will be returned. It is possible to build very complex workflows with data translations, sub-workflows, parallelisim and serialization, and multiple output paths.
```

Comment:
I'd be careful about this "parallelism and serialization" stuff; you don't *do* anything in Arcaflow to get parallelism, and you need to carefully set step dependencies to avoid it ... I don't think these words in this context prepare a reader for the details.

Reply:
But *not* doing something I would argue is an active choice by the workflow author to accept the default behavior, and they have therefore chosen to parallelize their steps. At this very early introductory point in the document, my intention is to simply outline what is possible without providing the details.

Reply:
That's kinda what I mean: the lack of action implies an "active choice", which is absolutely *not* obvious from this text. And in fact I think this phrasing implies we have parallelism and serialization "syntax" of some sort, which we don't. You can express dependencies between steps to ensure sequential operations, maybe?

Reply:
IMO this is too "early" in the documentation to be that specific. The reader consuming this getting started guide needs to get the gist clearly; they do not need all of the explicit details.

Reply:
Yes, and I would argue that the concepts of "parallelism" and "serialization" in this context are way too abstract to be useful to anyone, and would be better omitted. But I guess I'm done.